docs/en/connector-v2/sink/Datahub.md

```diff
@@ -6,6 +6,12 @@
 
 A sink plugin which use send message to datahub
 
+## Key features
+
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+
```

Comment:
Some places have blank lines and others don't

Reply:
done